### DISCUSSION TITLE
wails: command not found

### DISCUSSION THREAD

**Discussion by Blatman** [wailsapp/wails#1760] [General]
Just a wee note which may help a Mac user (I happen to be using 10.13 at the moment).
Got this bash error and normally it means that the command cannot be located by the OS via the $PATH however in my case the $PATH included /usr/local/go/bin as specified by the GO install instructions. 
The GO install also creates the workspace at /Users/USER/go which had subfolders bin and pkg.
The wails binary was installed in the /Users/USER/go/bin/ folder but was not available globally even though the $PATH was correct. 
What worked for me was to copy the wails executable into the /usr/local/go/bin/ folder and then it could be found via the $PATH (an alias didn't work). Now appears to work well and I can create/use app project folders in a src folder added to /Users/USER/go/ folder.
Just thinking that maybe Homebrew has at some stage upset a required linkage between /usr/local/go/bin/ and /Users/USER/go/bin/ so wails was not being seen despite the $PATH being OK.

**Comment by leaanthony** [comment]
This is discussed here https://wails.io/docs/guides/troubleshooting#the-wails-command-appears-to-be-missing. It's annoying the official instructions don't include that. Thanks for sharing

**Reply by Blatman** [reply]
Thanks for the link - hadn't got to that part of the docs yet - solved my Transport Security issue too.

BTW - on the troubleshooting page regarding the Application Not Valid - can be due to binary built for later OS and plist usually has a key for that  - 

	<key>LSMinimumSystemVersion</key>
	<string>10.15.0</string>

The Optimus example app shows the broken icon on 10.13 and although the plist doesn't include the MinVersion key the website of the dev states it is built for 10.15 or later and indeed will give a warning dialog message if run even without the plist key.

Cheers.
